Tasks/AzureAppServiceManageV0/operations/AzureResourceFilterUtils.ts

```diff
@@ -4,23 +4,17 @@ import { Resources } from 'azure-pipelines-tasks-azure-arm-rest/azure-arm-resour
 
 export class AzureResourceFilterUtils {
     public static async getResourceGroupName(endpoint: AzureEndpoint, resourceType: string, resourceName: string): Promise<string> {
-        var azureResources: Resources = new Resources(endpoint);
-        var filteredResources: Array<any> = await azureResources.getResources(resourceType, resourceName);
-        let resourceGroupName: string;
-        filteredResources = filteredResources.filter(
-            (obj, index) =>
-                filteredResources.findIndex((item) => item.id === obj.id) === index
-        );
-        if(!filteredResources || filteredResources.length == 0) {
+        const azureResources: Resources = new Resources(endpoint);
+        const resources: Array<any> = await azureResources.getResources(resourceType, resourceName);
+        const resourceGroupIds: Array<any> = [...new Set(resources.map(r => r.id as string))];
```

Comment:
I think that `Array<any>` is not needed here. The type of `resourceGroupIds` is inferred from the right side of assignment as `string[]` which is good as you can work with specific type rather than with `any` in the following code.

Reply:
@onetocny as suggested changed the assignment type from array to string